app/assets/javascripts/bastion/components/nutupane.factory.js

```diff
@@ -82,7 +82,7 @@ angular.module('Bastion.components').factory('Nutupane',
 
             self.loadParamsFromExistingTable = function (existingTable) {
                 _.extend(existingTable.params, params);
-                self.table.params = existingTable.params;
+                self.params = existingTable.params;
```

Comment:
@sjha4 can you explain to me the difference between `self.params` and `self.table.params`?  I've never understood why we have both.  /cc @ehelms

Reply:
There are places in katello I have seen table.params being used and places where params is used to pass to nutupane. 
In the load function we are sending params to the resource call, which to me means that the goal  is to merge everything into self.params  for everything to work.
https://github.com/Katello/bastion/blob/8c951343f34fc0934e03c9f3b62858016f6cd53a/app/assets/javascripts/bastion/components/nutupane.factory.js#L111